Makefile

```diff
@@ -1,38 +1,44 @@
-commit  := $(shell git rev-parse --short HEAD)
-tag     := $(shell git tag -l 'v*-rc*' --points-at HEAD)
-version := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
-artifactory_publish := $(shell if [[ -n "$(tag)" ]]; then echo release; else echo dev; fi)
+artifact_name   := company-accounts-library
+version         := "unversioned"
 
 .PHONY: all
 all: build
 
 .PHONY: clean
 clean:
-	mvn clean
+    mvn clean
 
 .PHONY: build
 build:
-	mvn compile
+    mvn compile
 
 .PHONY: test
 test: test-unit
 
 .PHONY: test-unit
 test-unit: clean
-	mvn test
+    mvn test
 
 .PHONY: package
 package:
-	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
-	mvn package -DskipTests=true
+ifndef version
+    $(error No version given. Aborting)
+endif
+    $(info Packaging version: $(version))
+    mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
+    mvn package -DskipTests=true
 
 .PHONY: dist
 dist: clean package
 
 .PHONY: publish
 publish:
-	mvn jar:jar deploy:deploy -DpublishRepo=$(artifactory_publish)
+    mvn jar:jar deploy:deploy
 
 .PHONY: sonar
 sonar:
-	mvn sonar:sonar
+    mvn sonar:sonar
+
+.PHONY: sonar-pr-analysis
+sonar-pr-analysis:
+    mvn sonar:sonar -P sonar-pr-analysis
```

Comment:
You'll need to extend a ch parent pom if you want to use this profile

Reply:
I'll merge #10 then